src/components/animated/_colored-on-hover.less

```diff
@@ -0,0 +1,108 @@
+@keyframes fonnt {
+	0% {
+		color: black;
+	}
+	100% {
+		color: cornsilk;
+		text-shadow: 0px 0px 20px white;
+		box-shadow: inset 0px 0px 5px cornsilk;
+	}
+}
+@keyframes fonntrev {
+	0% {
+		color: black;
+	}
+	100% {
+		color: black;
+		box-shadow: inset 0px 0px 5px black;
+	}
+}
+.sbtn.colored-on-hover-btn {
+	position: relative;
+	padding: 0px;
+	color: black;
+	height: 50px;
+	width: 200px;
+	background-color: transparent;
+	cursor: pointer;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	outline: none;
+	&:hover {
+		animation: fonnt 0.25s ease-in forwards;
+	}
+	.bluebtn{
+		&:hover {
+			animation: fonnt 0.25s ease-in forwards;
+			background-color: blue;
+		}
+		border: 3px double blue;
```

Comment:
I don't think using an animation is necessary. Just change the background color on hover and if necessary the color of the button as well, and to make the transition seamless add a transition rule with a duration.

In addition I see you have two borders for the buttons but we don't want that. Please remove the additional button. The button should look exactly like the outline button at first, just on hover the background color and the color of the text changes. So, please refer to the code for outline button for help.

Also, the block code inside the color buttons is the same except for the colors used so create a mixin with this block of code that takes a color as a parameter and call it from each of the buttons. Also make sure to use the project's color variables instead of literal colors. For example instead of `blue` use `@blue`.

Reply:
Okay I would make the necessary changes soon